### ISSUE TITLE
Option to return Chrome versions for Chromium-based browsers

### ISSUE THREAD

**Issue by ylemkimon** [browserslist/browserslist#508]
Due to insufficient compatibility data, most tools ignore browsers unpopular in US, such as UC Browser and QQ Browser. (e.g., https://github.com/babel/babel/pull/8552) But as far as I understand, this is the very reason Browserslist is created. Fortunately, most of them are based on Chromium and compatibility is almost identical to the version of Chromium they are based on. So an option to return corresponding Chrome versions for those browsers would be useful and encourage people to support them.

**Comment by ai** [comment]
We already have `mobileToDesktop` option. We can add something like `normalize`.

Do you want to send PR?

**Comment by ylemkimon** [comment]
@ai Yes, I would like to send a PR. One question is should I implement this as a query or an option? The former seems to be backward compatible but may break if the version doesn't match, whereas the latter needs to update all the tools.

**Comment by ai** [comment]
As an option, because it if a feature for a tool (like Babel), not a user.

It will be much easier for us to convince a few tools to use the new option, than change user’s configs.

**Comment by ai** [comment]
@JLHwung will it be a useful option for Babel?
